StyleCop.Analyzers/StyleCop.Analyzers/SpacingRules/SA1013ClosingCurlyBracketsMustBeSpacedCorrectly.cs

```diff
@@ -107,10 +108,17 @@ private void HandleCloseBraceToken(SyntaxTreeAnalysisContext context, SyntaxToke
                 precedesSpecialCharacter = false;
             }
 
+            var parent = token.Parent as InterpolationSyntax;
+            if (parent != null)
+            {
+                // Don't report for interpolation string inlets
+                return;
+            }
+
             if (!firstInLine && !precededBySpace)
             {
                 // Closing curly bracket must{} be {preceded} by a space.
-                context.ReportDiagnostic(Diagnostic.Create(Descriptor, token.GetLocation(), string.Empty, "preceded"));
+                context.ReportDiagnostic(Diagnostic.Create(Descriptor, token.GetLocation(), string.Empty, $"pre{5}ceded"));
```

Comment:
This is what happens if you dont look at the diff before you commit >.< I was looking at the syntax visualizer to see how the class I was looking for was called.